### DISCUSSION TITLE
Timing of March Update

### DISCUSSION THREAD

**Discussion by wchest** [mozilla/hubs#4092] [Hubs Cloud Questions] [answered]
We have a notice in our Hubs Cloud instance admin that says: 
```
 A future update will impact your Hubs Cloud deployment ⚠️

We will release an updated client interface for Hubs Cloud after March 25, 2021. This update significantly changes the way Hubs looks, but all functionality will remain the same. For more information visit Hubs Docs: Hubs Cloud Customizing Look and Feel.
```
Is there any way to know when this update will take place? We have a number of events in the coming month that will use Hubs and it would be good to know if our stack will be going offline at some point during that time.

Thanks!

**Comment by johnshaughnessy** [comment, marked as answer]
We have been testing the next release for the last few weeks. It's in the final stage, which is when we fix issues that came up during testing. We'll share a release date when we have one and announce it on discord and twitter. We will share an exact date at least a week in advance. Thank you for your patience.

**Reply by rawnsley** [reply]
@johnshaughnessy What branch are you using for this? It would be useful to pull in the current version so I can merge my custom client changes even if there will be some small changes afterwards when you merge it into _hubs-cloud_.

**Reply by blairmacintyre** [reply]
Indeed ... inquiring minds want to know, so we can start making sure our stuff works without a 1 week crunch

**Reply by wchest** [reply]
Thanks for the update! We'll keep an eye out for the announcement.

**Reply by johnshaughnessy** [reply]
> @johnshaughnessy What branch are you using for this? It would be useful to pull in the current version so I can merge my custom client changes even if there will be some small changes afterwards when you merge it into _hubs-cloud_.

The `qa-test` branches reflect the code being tested for deployment in the next Hubs Cloud release.
Hubs: https://github.com/mozilla/hubs/tree/qa-test
Reticulum: https://github.com/mozilla/reticulum/tree/qa-test
Spoke: https://github.com/mozilla/spoke/tree/qa-test

Note that if you want to "freeze" your client in place and update it at a time of your choosing, you can deploy a custom client and then undeploy it once you are ready to get the update. See https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html for details.

**Reply by terhell23** [reply]
We are hosting an event on the 21 and 22 of April with mozilla hubs. If the update would be released on the 21th of  april we would have a lot of trouble. Do you now yet if the update is released before or after that date?
